addons_rksv/pos_rksv/static/src/js/screens.js

```diff
@@ -206,7 +211,17 @@ odoo.define('pos_rksv.screens', function (require) {
             this.events['click .export_crypt'] = 'export_crypt';
             this.events['click .start_receipt_set_valid'] = 'start_receipt_set_valid';
         },
+        willStart: function() {
```

Comment:
willStart is in between init and start - start will get called as soon as the deferred does resolv.

It is in the base widget class - odoo web core framework